tests/plugins.t

```diff
@@ -53,13 +53,10 @@ my ( $test_eH_gID, $test_eH_gToken ) =
 is( $test_eH_gID,    $eH_gID,    'eHentai search test 1/2' );
 is( $test_eH_gToken, $eH_gToken, 'eHentai search test 2/2' );
 
-my $eH_tags =
-  "parody:touhou project, character:hong meiling, character:marisa kirisame, character:reimu hakurei, character:sanae kochiya, character:youmu konpaku, group:handful happiness, artist:nanahara fuyuki, artbook, full color, category:non-h";
-
-my ( $test_eH_tags, $test_eH_title ) = LANraragi::Plugin::Metadata::EHentai::get_tags_from_EH( $ua, $eH_gID, $eH_gToken );
-
-is( split( ", ", $test_eH_tags ), split( ", ", $eH_tags ), 'eHentai API Tag retrieval test' );
-is( $test_eH_title,
+my $test_eH_json = LANraragi::Plugin::Metadata::EHentai::get_tags_from_EH( $ua, $eH_gID, $eH_gToken );
+ok( exists $test_eH_json->{gmetadata}, 'gmetadata exists' );
+isa_ok( $test_eH_json->{gmetadata}, 'ARRAY', 'type of gmetadata' );
```

Comment:
I'm wondering if it'd be worth it to also check `gmetadata[0]{tags}` here with a `cmp_bag`: Checking the title is probably already a pretty good way of making sure the live API still works, so it's up to you if you want to do it. 😀

Reply:
I could, but I think this test is better done inside the `EHentai.t` specific module.

`plugin.t` is very similar to an integration test and it calls many modules.
 Adding here a "cmp_bag" test for any plugin used, will result in cluttering the code.
Besides I should have removed also the title because, like tags, it can change any moment breaking the test. Its better to test them using the "more stable" samples. 

Normally in this type of tests I check the response codes or that the needed fields are present. So I can be sure the API haven't changed.

Reply:
Np, fair enough.